ape_etherscan/config.py

```diff
@@ -1,12 +1,34 @@
+from typing import Optional
+
 from ape.api.config import PluginConfig
+from pydantic import AnyHttpUrl, model_validator
+from pydantic_settings import SettingsConfigDict
+
+
+class NetworkConfig(PluginConfig):
```

Comment:
Can we mention how to configure these in the README, like you did in the description?
I think right now, the README mentions "all the supported networks". Should be updated to mention custom nets also work

Reply:
Updated README.  Link to the docs might need to be updated after your work is documented.  I made a guess.